client/src/main/java/app/pizzeria/client/service/PalmettoNotificationService.java

```diff
@@ -0,0 +1,26 @@
+package app.pizzeria.client.service;
+
+import app.pizzeria.common.model.OrderDto;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
should be named as @Service, as it contains business logic

Reply:
done, thank you!